### ISSUE TITLE
Becareful, My id was banned !

### ISSUE THREAD

**Issue by songwutk** [grishka/Houseclub#225]
**Describe the bug**
While I listen the  topic, the sound stop. I refresh screen, that message show "your account has been temporarily suspended. Please contact us at suspension@joinclubhouse.com"

**Comment by Ahmad90kw** [comment]
Did you have a fake name or pic?
I'm still using the app and it is working perfect
Go read the guidelines if u have a fake name or a fake pic or if u violate a bunch of rules then they will suspend you

**Comment by ByDSH20** [comment]
I am using version 1.0.9
Name and photo are real
There was still no ban
I have been in this social network since February 23

**Comment by pickyzz** [comment]
My backup ID got banned too. Even I use real name and profile picture.

About 12 hours ago.

**Comment by XtratoS** [comment]
> I am using version 1.0.9
> Name and photo are real
> There was still no ban
> I have been in this social network since February 23

@ByDSH20 There's no version 1.0.9

**Comment by XtratoS** [comment]
I have not violated any rules, I have my real name, my real photo and I used an iPhone to create the account and link it to Instagram. I still got banned.

**Comment by ByDSH20** [comment]
> > Я использую версию 1.0.9. 
> > Имя и фото настоящие. 
> > Бана еще не было. 
> > Я в этой соцсети с 23 февраля.
> 
> @ ByDSH20 Нет версии 1.0.9

I found it somewhere on this site
Both search and invites appeared 
![Screenshot_20210302-090904_Houseclub](https://user-images.githubusercontent.com/79893198/109606020-14f5c180-7b37-11eb-91d4-9aa222f00f58.jpg)

**Comment by YousfAsbhi** [comment]
@ByDSH20 I couldn't find it in the releases section, can you please send a link 

> > > Я использую версию 1.0.9.
> > > Имя и фото настоящие.
> > > Бана еще не было.
> > > Я в этой соцсети с 23 февраля.
> > 
> > 
> > @ ByDSH20 Нет версии 1.0.9
> 
> I found it somewhere on this site
> Both search and invites appeared
> ![Screenshot_20210302-090904_Houseclub](https://user-images.githubusercontent.com/79893198/109606020-14f5c180-7b37-11eb-91d4-9aa222f00f58.jpg)

**Comment by zverev-iv** [comment]
Me banned to. Used This app only. Real Photo and Name. No spamming or rule violation.

**Comment by onuraydin** [comment]
Same here, I'm banned too. I was just listening, never interacted with anyone.

**Comment by Freedomby420** [comment]
Anyone have a solution? Tell me.

**Comment by itszakjo** [comment]
Well. After investigation. I assume they ban Android accounts using the DEVICE_ID , yeah device id will tell if your device is iOS or android.

**Comment by mkljczk** [comment]
Guess we will have to tell Clubhouse that we only use Houseclub as a temporary way to access the app, it has no shady features or stuff that don’t want apps to have (like audio recording)

**Comment by onuraydin** [comment]
https://clubhouseapp.zendesk.com/hc/en-us/requests/new you can request, but probably clubhouse is banning every android account now. So, it's not safe to use this app.

**Comment by mkljczk** [comment]
I sent them a request mentioning my personal feelings and the fact that we are basically just users waiting for official Android version of Clubhouse.

**Comment by wsboner** [comment]
> @ByDSH20 I couldn't find it in the releases section, can you please send a link
> 
> > > > Я использую версию 1.0.9.
> > > > Имя и фото настоящие.
> > > > Бана еще не было.
> > > > Я в этой соцсети с 23 февраля.
> > > 
> > > 
> > > @ ByDSH20 Нет версии 1.0.9
> > 
> > 
> > I found it somewhere on this site
> > Both search and invites appeared
> > ![Screenshot_20210302-090904_Houseclub](https://user-images.githubusercontent.com/79893198/109606020-14f5c180-7b37-11eb-91d4-9aa222f00f58.jpg)
 
Did u find version 1.0.9 ?

**Comment by ByDSH20** [comment]
> > @ ByDSH20 Я не нашел в разделе релизов, не могли бы вы прислать ссылку
> > > > > Я использую версию 1.0.9. 
> > > > > Имя и фото настоящие. 
> > > > > Бана еще не было. 
> > > > > Я в этой соцсети с 23 февраля.
> > > > 
> > > > 
> > > > @ ByDSH20 Нет версии 1.0.9
> > > 
> > > 
> > > Я нашел это где-то на этом сайте. 
> > > Появились и поиск, и инвайты.
> > > ![Screenshot_20210302-090904_Houseclub](https://user-images.githubusercontent.com/79893198/109606020-14f5c180-7b37-11eb-91d4-9aa222f00f58.jpg)
> 
> Вы нашли версию 1.0.9?

 https://github.com/bit-world/Houseclub/releases/tag/1.0.9
This is not from this developer (sort of)

**Comment by 0x366** [comment]
What happens with account, that was banned? Others can't find it anymore in search?  Could someone with banned account share their nick to check?

**Comment by freemin7** [comment]
I'm using https://github.com/grishka/Houseclub/releases/tag/1.0.8 however i am not banned. I have and never had the search bar or the add people symbol

**Comment by jnubian** [comment]
I'm using 1.0.8 and banned
This shows up in notifications. 
![Screenshot_20210301-180005_Houseclub](https://user-images.githubusercontent.com/63024646/109692774-e7207500-7b56-11eb-80a6-ccbb6bc49804.jpg)

But people can still follow my account. I cannot update my bio or join rooms to listen.

**Comment by Mayrakis** [comment]
I think you just reported by iOS user, iOS have report function.

**Comment by Freedomby420** [comment]
Good grief.

**Comment by hannyhashmi** [comment]
Is there a way to spoof iOS ID using a rooted android phone?

**Comment by caasn** [comment]
If my russian is good enough that's what the guys here are thinking of. I just don't know when it's implemented.

**Comment by Ahmad90kw** [comment]
Could it be if someone is using a wifi network or a cellular network? I have never used a cellular network to log in and I am not banned or anything

**Comment by caasn** [comment]
This app is in Google Play. It says it's version 1.1.0.9 and it's working as far as I can see. Is it trustworthy? https://play.google.com/store/apps/details?id=ru.turbominer.houseclub&hl=de&gl=US

**Comment by mkljczk** [comment]
@caasn not so trustworthy, as he used iOS screenshot of OG app in Play Store

**Comment by jnubian** [comment]
@caasn can you verify that it is working on your own device? 
Misinformation leads to lies being culturally accepted as truths.

**Comment by caasn** [comment]
It's working on my device.

**Comment by jnubian** [comment]
@caasn thank you for confirming this. 

My account was banned using this app.
Or maybe the device ID is banned.

 It is till banned after using the app from the link you provided. I can see rooms, get new followers. But get error message when trying to listen.

Was your account banned prior? 

The app in playstore seems to have more functionality as invites are included.

**Comment by caasn** [comment]
I don't see other funtions as in the 1.0.9 version. I was banned with an account with this personal icon, now I'm using another one with online sms that's not banned yet. I'm mostly in rooms with german speakers and I'm not talking, maybe you're not banned so quickly in those rooms. I guess I was banned by a user before as Mayrakis said before.

**Comment by Mayrakis** [comment]
> > > @ ByDSH20 Я не нашел в разделе релизов, не могли бы вы прислать ссылку
> > > > > > Я использую версию 1.0.9.
> > > > > > Имя и фото настоящие.
> > > > > > Бана еще не было.
> > > > > > Я в этой соцсети с 23 февраля.
> > > > > 
> > > > > 
> > > > > @ ByDSH20 Нет версии 1.0.9
> > > > 
> > > > 
> > > > Я нашел это где-то на этом сайте.
> > > > Появились и поиск, и инвайты.
> > > > ![Screenshot_20210302-090904_Houseclub](https://user-images.githubusercontent.com/79893198/109606020-14f5c180-7b37-11eb-91d4-9aa222f00f58.jpg)
> > 
> > 
> > Вы нашли версию 1.0.9?
> 
> https://github.com/bit-world/Houseclub/releases/tag/1.0.9
> This is not from this developer (sort of)

Invite function in this version is working?

**Comment by caasn** [comment]
I get the information that I don't have invites so far, but I was invited using this version, if that answers your question.

**Comment by writerolga** [comment]
А эту версию сборки приложения планируется дообновить до дополнительных функций (раздача инвайтов, строка поиска) или нет?

**Comment by XtratoS** [comment]
> А эту версию сборки приложения планируется дообновить до дополнительных функций (раздача инвайтов, строка поиска) или нет?

No, the original author has no plan to do so if you read the README
